app/views/groups/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="splash2">
+  <div class="top-strip2">
+    <%= button_to 'Sign Out', destroy_user_session_path, class:"signout-link", method: :delete %>
+    <h2>CATEGORIES</h2>
+  </div>
+<% @groups.each do |group| %>
+  <div class="group-container">
+    <div class="group-element">
+      <img src="<%= group.icon %>" class="no-underline group-image">
+    </div>
+    <%= link_to group_deals_path(group), class:"group-link group-element" do %>
+      <p><%= group.name %></p>
+    <% end %>
+    <p class="group-element"><%= number_to_currency(group.deals.where(author: current_user).sum(:amount)) %></p>
+  </div>
+<% end %>
+<%= link_to 'Add a New Category', new_group_path, class:'new-category-link'%>
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 take in mind that it is a mobile web app, when I checked with a width of 375px the UI should be similar to the original design, i.e. the button should cover the width. 👇🏽

![image](https://github.com/ehmaddd/BudgetApp/assets/58642949/45ec360b-2f8f-41ba-947a-40235cbd8689)